blazor/datagrid/filter-menu.md

```diff
@@ -223,4 +223,106 @@ The following screenshot represents CheckBox filter
 
 ### Limitations of using different filter types in different columns
 
-The different filter types such as Excel, Menu, and Checkbox can be defined in different columns of the same Grid. However, you cannot use these filter types along with filterBar type (default filter type). Because the filterbar type requires UI level changes with other filter types. For all other filter types, icons will be rendered in the column header.
\ No newline at end of file
+The different filter types such as Excel, Menu, and Checkbox can be defined in different columns of the same Grid. However, you cannot use these filter types along with filterBar type (default filter type). Because the filterbar type requires UI level changes with other filter types. For all other filter types, icons will be rendered in the column header.
```

Comment:
The different filter types such as Excel, Menu, and Checkbox can be defined in different columns of the same Grid. However, you can't use these filter types along with the filterBar type (default filter type) because the filterbar type requires UI level changes with other filter types. Icons will be rendered in the column header for all other filter types.